src/app/child-dev-project/attendance/add-day-attendance/roll-call/roll-call.component.ts

```diff
@@ -98,6 +116,16 @@ export class RollCallComponent implements OnInit {
     }
   }
 
+  private sortParticipants() {
+    if (!this.sortParticipantsBy) {
+      return;
+    }
+
+    this.entries.sort((a, b) => a.child.name.localeCompare(b.child.name));
+    // also sort the participants in the Note entity itself for display in details view later
+    this.eventEntity.children = this.entries.map((e) => e.child.getId());
+  }
+
```

Comment:
Am I missing something here or is the actual value of `this.sortParticipantsBy` ignored (and the entities are only sorted by name?)

Reply:
damn it ... good thing I put this up for review. You are absolutely right :sweat_smile: